appveyor-gn.yml

```diff
@@ -32,13 +32,18 @@ build_script:
   - gn gen out/Default "--args=import(\"//electron/build/args/%gn_args%.gn\")"
   - ninja -C out/Default electron:electron_app
 test_script:
-  - ninja -C out/Default electron/build/node:headers
+  - ninja -C out/Default third_party/electron_node:headers
   - ps: $env:npm_config_nodedir="$pwd/out/Default/gen/node_headers"
   - ps: $env:npm_config_msvs_version="2017"
+  - ps: >-
+      if ($env:gn_args -eq 'testing') {
```

Comment:
We shouldn't even try to run tests in debug configuration. Once it is done, this condition should be removed.
cc @jkleinsc